enterprise/api/v1/serializers.py

```diff
@@ -94,8 +94,15 @@ class Meta:
             'tertiary_color',
         )
 
+    enterprise_customer = serializers.SerializerMethodField()
```

Comment:
This had to be added to prevent `TypeError: Object of type 'UUID' is not JSON serializable`